app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApplicationController
+      def create
+        book_suggestion = BookSuggestion.new(book_suggestion_params)
+        if book_suggestion.save
+          render json: book_suggestion, status: :created
+        else
+          render json: book_suggestion.errors, status: :unprocessable_entity
+        end
+      end
+
+      private
+
+      def book_suggestion_params
+        params.require(:book_suggestion).permit(:title, :author, :link)
```

Comment:
We are never accepting user_id for example, so you will not be able to relationate User with a book_suggestion